docker/Dockerfile.llvm

```diff
@@ -34,8 +34,10 @@ RUN apt update \
                          --slave /usr/bin/g++ g++ /usr/bin/g++-8      \
   && cp /usr/lib/gcc/x86_64-linux-gnu/8/plugin/include/plugin-api.h /usr/local/include
 
-RUN curl -L --output /tmp/cmake.tar.gz \
-  https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0-linux-x86_64.tar.gz \
+ARG ARCH
+RUN ARCH=$(uname -m) \
+  && curl -L --output /tmp/cmake.tar.gz \
+  https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0-linux-${ARCH}.tar.gz \
```

Comment:
Beware that `x86_64` is also used few lines above in this Dockerfile.

Reply:
Thanks for catching it. I searched all dockerfiles for x86_64 and replaced them with $(uname -m) in latest iteration.